explorer/nodewatch/nodewatch/__init__.py

```diff
@@ -39,12 +39,16 @@ def nem_summary():  # pylint: disable=unused-variable
 		template_name, context = nem_routes_facade.html_summary()
 		return render_template(template_name, **context)
 
-	@app.route('/nem/chart/height')
-	def nem_chart_height():  # pylint: disable=unused-variable
+	@app.route('/api/nem/nodes')
+	def api_nem_nodes():  # pylint: disable=unused-variable
+		return nem_routes_facade.json_nodes(1)
+
+	@app.route('/api/nem/chart/height')
+	def api_nem_chart_height():  # pylint: disable=unused-variable
 		return nem_routes_facade.json_height_chart_with_metadata()
 
-	@app.route('/nem/height')
-	def nem_height():  # pylint: disable=unused-variable
+	@app.route('/api/nem/height')
+	def api_nem_height():  # pylint: disable=unused-variable
 		return nem_routes_facade.json_height()
```

Comment:
should we keep rendered vs non-rendered apis in some kind of separate groups/regions?

Reply:
it should be already

/nem
/api/nem
/symbol
/api/symbol

do you mean to swap /api/nem and /symbol ?

Reply:
I thought about completely separating, as we have:
/nem/harvesters
/nem/nodes
...

/symbol/voters
/symbol/harvesters
...

keep apis completely separate down in
/api/nem/foo
/api/symbol/foo

not sure if good idea, just an idea

Reply:
i'd rather separate nem / symbol and then api within that